### PR TITLE
WMS ID: #11732

### PR DESCRIPTION
**Pull request by JBAnderson5**
# Add here your WMS ID: 11732

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by ramonamagadan18 — COMMENTED** [review]
Hi @JBAnderson5! I see that you have updated two folders. Are they for the same workshop? Thank you!

**Comment by JBAnderson5** [comment]
Hi @ramonamagadan18 
TLDR: I updated the PR so only the generative-ai-agent-oda folder is being updated in this PR

Both folders are labs owned by my team that we work on. There were some merge conflicts in the ATOM lab folder that I resolved. There were some additional expansion workshops for the ATOM lab whose changes got into this PR. None of the changed/added files in the ATOM workshop are referenced in the manifest.json files, so nothing in this lab would have changed. I stashed those changes and will include them in a different PR for the ATOM lab
